app/controllers/application_controller/ci_processing.rb

```diff
@@ -517,7 +517,7 @@ def live_migrate_form_fields
 
   def live_migrate_vm
     assert_privileges("instance_live_migrate")
-    @record = VmOrTemplate.find_by(:id => params[:id])
+    @record = find_by_id_filtered(VmOrTemplate, params[:id])
```

Comment:
`find_by_id_filtered` method was renamed to `find_record_with_rbac` (https://github.com/ManageIQ/manageiq-ui-classic/pull/779/commits/dad34557eb21d21317126a51f30f9384197c0ea4)